backend/core/src/map-layers/dto/map-layer.dto.ts

```diff
@@ -0,0 +1,12 @@
+import { Expose } from "class-transformer"
+
+export class MapLayerDto {
+  @Expose()
```

Comment:
I think these fields should include other decorators such as isString(), isUUID(), and ApiProperty(). There's some good examples in other dtos around how to treat id, name, and jurisdictionId.